src/main/java/core/basesyntax/impl/ReadingCsvFileImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.impl;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package statement.